Fix: prevent ripple effect on slider cell in BottomSheet and disable it completely on iOS (#2023)
wordpress-mobile/gutenberg-mobile: RELEASE-NOTES.txt

1.26.0
------

> * Disable ripple effect on iOS in BottomSheet
> * Disable ripple effect for Slider in BottomSheet on iOS and Android

Reviewer: We usually prefix with [Android] or [iOS] for platform specific changes. In this case I would go with something like this:

```suggestion
* [iOS] Disable ripple effect in all BottomSheet's controls.
* [Android] Disable ripple effect for Slider control
```
